src/SolidStateDetectors.jl

```diff
@@ -43,7 +43,7 @@ using .ConstructiveSolidGeometry:
             ConfigFileError, _parse_value,
             LengthQuantity, AngleQuantity
         
-import .ConstructiveSolidGeometry: sample, to_internal_units, from_internal_units
+import .ConstructiveSolidGeometry: sample, to_internal_units, from_internal_units, get_scale

```

Comment:
Why are you importing `get_scale`? Do you add methods to `get_scale` in SSD outside of CSG?

Reply:
It is used outside of CSG in the plot recipe for SolidStateDetector. Should have put it in the `using .ConstructiveSolidGeometry:` above this line. Changed.